portfolio/src/main/webapp/index.js

```diff
@@ -13,17 +13,21 @@
 // limitations under the License.
 
 "use strict";
-import {setToNone} from "./utils.js";
 import {SlideShow} from "./slideshow.js";
 import {Tab} from "./tab.js";
+import {Utils} from "./utils.js";
+
+const tab = new Tab();
+const slideShow = new SlideShow();
 
 window.onload = function() {
-  // Allow all imported functions to be in the scope of "window".
-  window.tab = new Tab();
-  window.slideShow = new SlideShow();
-  
+  // Allow all imported functions to be in the scope of "window" in 
+  // window.onload.
+  window.tab = tab;
+  window.slideShow = slideShow;
+
   // Set up default tab and slideshow.
-  tab.displayDefaultSection();
-  setToNone("slide");
-  slideShow.showSlide(slideShow.slideIndex);
+  window.tab.displayDefaultSection();
+  Utils.setToNone("slide");
+  window.slideShow.showSlide(window.slideShow.currentSlideIndex_);
```

Comment:
Don't access private variables outside of their class. Either create a getter function or just display the 0th slide.